packages/lumx-react/.storybook/postcss.config.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable global-require, import/no-commonjs, import/unambiguous */
+module.exports = {
```

Comment:
Why not use the `configs/postcss.config.js` directly instead of copying?

Reply:
I actually took it from the configurations from the `site-demo` package, I did not see that there was this configuration on the root level. Maybe the one from the site demo should be changed as well?

Reply:
Ah yes I forgot I did a symbolic link from `/configs/postcss.config.js` to `/packages/site-demo/postcss.config.js`.
Indeed it would be simpler just to reference the original config path in both webpack/storybook configurations.